pkg/webui/components/data-sheet/story.js

```diff
@@ -46,6 +46,13 @@ const testData = [
     header: 'Activation Info',
     items: [
       { key: 'Device EUI', value: '1212121212', type: 'byte', sensitive: false },
+      {
```

Comment:
Currently, storybook is broken. Please add:
```js
  window.__ttn_config__ = {
    APP_CONFIG: {
      documentation_base_url: 'https://thethingsstack.io',
    },
  }
```

to `config/storybook/preview-head.html`